BuildLoader.py

```diff
@@ -393,6 +400,15 @@ def update_fit_table (self):
 
 		else :
 			addr = fit_address.value + (num_fit_entries + 1) * 16
+			if self._board.ACM3_SIZE > 0:
```

Comment:
These 3lines must be located before line#402 "addr = fit_address.value + (num_fit_entries + 1) * 16".